core/src/main/java/org/infinispan/configuration/cache/IndexWriterConfiguration.java

```diff
@@ -14,9 +14,9 @@ public class IndexWriterConfiguration extends ConfigurationElement<IndexWriterCo
    public static final AttributeDefinition<Integer> INDEX_THREAD_POOL_SIZE =
          AttributeDefinition.builder(Attribute.THREAD_POOL_SIZE, 1, Integer.class).immutable().build();
    public static final AttributeDefinition<Integer> INDEX_QUEUE_COUNT =
-         AttributeDefinition.builder(Attribute.QUEUE_COUNT, 1, Integer.class).immutable().build();
+         AttributeDefinition.builder(Attribute.QUEUE_COUNT, 16, Integer.class).immutable().build();
    public static final AttributeDefinition<Integer> INDEX_QUEUE_SIZE =
-         AttributeDefinition.builder(Attribute.QUEUE_SIZE, null, Integer.class).immutable().build();
+         AttributeDefinition.builder(Attribute.QUEUE_SIZE, 2_000, Integer.class).immutable().build();
```

Comment:
Careful with conflicts, @wburns 's PR assumes the default is Hibernate Search's (1000).

Reply:
Yes we're going to adapt the second PR we merge.

Reply:
Yeah, if we always supply a default then my if can just be removed 👍